framework/src/vectorpostprocessors/PointSamplerBase.C

```diff
@@ -159,10 +168,49 @@ PointSamplerBase::transferPointsVector(std::vector<Point> && points)
 const Elem *
 PointSamplerBase::getLocalElemContainingPoint(const Point & p)
 {
-  const Elem * elem = (*_pl)(p);
+  if (_discontinuous_variables)
```

Comment:
```suggestion
  const Elem * elem = nullptr;
  if (_discontinuous_variables)
```

This'll clean up some of the logic below